p11-kit/rpc-transport.c

```diff
@@ -1145,6 +1146,89 @@ rpc_unix_init (const char *remote,
 
 #endif /* OS_UNIX */
 
+#ifdef HAVE_VSOCK
+#include <linux/vm_sockets.h>
+#include <vsock.h>
+
+typedef struct {
+	p11_rpc_transport base;
+	struct sockaddr_vm sa;
+} rpc_vsock;
+
+static CK_RV
+rpc_vsock_connect (p11_rpc_client_vtable *vtable,
+		   void *init_reserved)
+{
+	rpc_unix *run = (rpc_unix *)vtable;
```

Comment:
Why not use `rpc_vsock`?

Reply:
Because it escaped my search/replace when I cut and pasted the unix code and modified it. Will fix.

Reply:
Fixed.